tools/shell-helpers/copy-folder.js

```diff
@@ -0,0 +1,24 @@
+const {
+  readdirSync,
+  mkdirSync,
+  lstatSync,
+  copyFileSync,
+  rmdirSync,
+} = require('fs');
+const { join } = require('path');
+const { argv } = require('process');
+
+function copyFolder(from, to) {
+  rmdirSync(to, { recursive: true });
+  mkdirSync(to);
+  for (const element of readdirSync(from)) {
+    if (lstatSync(join(from, element)).isFile()) {
+      copyFileSync(join(from, element), join(to, element));
+    } else {
+      copyFolder(join(from, element), join(to, element));
+    }
+  }
+}
+
+const [from, to] = argv.slice(2);
+copyFolder(from, to);
```

Comment:
If you import the function from this file this will get called so you might call it twice!

http://www.ecma-international.org/ecma-262/6.0/#sec-abstract-module-records
![image](https://user-images.githubusercontent.com/11156362/91688316-03b90180-eb62-11ea-840d-4d58c50a41fe.png)



There are two solutions. 
1. split it up in a CLI entry point file that imports the method and calls it so when you import the method, it doesn't get called
2. wrap the method call in an `if (!require.main?.filename)  { ... }`

Reply:
I'm not really sure what you're referring to. You mean if I `require('copy-folder.js')`? The `copyFolder` function isn't exported since I didn't think that we would need it outside of CLI so it shouldn't be a problem if I understand correctly. I could export it and wrap the call if you think that we'll need it though.

Reply:
ah sorry, I've overseen this you are right! If you would export it then it would be necessary.

To be honest I would export it if we need it somewhere else